src/trio/_core/_run.py

```diff
@@ -348,7 +348,7 @@ class CancelStatus:
     # become effectively cancelled when they do, unless scope.shield
     # is True.  May be None (for the outermost CancelStatus in a call
     # to trio.run(), briefly during TaskStatus.started(), or during
-    # recovery from mis-nesting of cancel scopes).
+    # recovery from miss-nesting of cancel scopes).
```

Comment:
this, and the other "miss-nesting"s, are incorrect fixes. https://en.m.wiktionary.org/wiki/misnested

Reply:
Fixed with 4431971